### DISCUSSION TITLE
administrator page, not all streaming is seen

### DISCUSSION THREAD

**Discussion by masterfm23** [AzuraCast/AzuraCast#6029] [Q&A]
What happened?
https://radios.lamaster.es:8000/

Since its update, the administration page does not show all the streaming that are working on that same port.
![WhatsApp Image 2023-01-17 at 10 34 14 AM](https://user-images.githubusercontent.com/122870464/212865101-e4870761-7b34-4f27-9c63-9f6b6cbb87c5.jpeg)

**Comment by JoseEspadas** [comment]
How to hide this page and nobody can see it?
https://radios.lamaster.es:8000/
